src/test/scala/com/cognite/sdk/scala/v1/ClientTest.scala

```diff
@@ -329,17 +345,27 @@ class ClientTest extends SdkTestSpec with OptionValues {
       StatusCode.ServiceUnavailable, "", Seq(Header("content-type", "text/html; charset=UTF-8")))
     val badRequestBytes: Response[Array[Byte]]  = Response("".getBytes("utf-8"),
       StatusCode.ServiceUnavailable, "", Seq(Header("content-type", "unknown")))
-    val badRequestBackendStub = SttpBackendStub.synchronous
+    val badRequestBackendStub1 = SttpBackendStub(implicitly[MonadAsyncError[IO]])
       .whenAnyRequest
       .thenRespondCyclicResponses(badGatewayResponseBytes,
         unavailableResponseBytes,
         serverErrorBytes,
         serverErrorHtmlBytes,
         badRequestBytes,
         protobufResponse)
-    val points = retryingClient(badRequestBackendStub)
-      .dataPoints
-      .queryById(123, Instant.ofEpochMilli(1546300800000L), Instant.ofEpochMilli(1546900000000L), None)
+    val client2 = new GenericClient[IO]("scala-sdk-test",
+      projectName,
+      "https://www.cognite.com/nowhere-at-all",
+      ApiKeyAuth("irrelevant", Some("randomproject"))
+    )(
+      implicitly,
+      new RetryingBackend[IO, Any](
+        badRequestBackendStub1,
+        initialRetryDelay = 1.millis,
+        maxRetryDelay = 2.millis)
+    )
+    val points = client2.dataPoints.queryById(123, Instant.ofEpochMilli(1546300800000L), Instant.ofEpochMilli(1546900000000L), None)
+      .unsafeRunSync()
```

Comment:
Did this remain from testing?

Reply:
It does the same thing as the old test? The results are from the "cyclic responses" above.